JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -35,7 +43,54 @@ FlowSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is use
 FeedbackSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is used to provide the control system with measurements of physical quantities necessary to close control loops.,https://www.theworldavatar.com/kg/ontodevice/,Feedback sensor
 ProximitySensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that takes a continuous distance reading and returns a corresponding analog voltage.,https://www.theworldavatar.com/kg/ontodevice/,Proximity sensor
 SashOpeningSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that measures the sash opening of an object such as a fumehood.,https://www.theworldavatar.com/kg/ontodevice/,Sash opening sensor
+ConductivitySensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that measures the conductivity of a material.,https://www.theworldavatar.com/kg/ontodevice/,Conductivity sensor
+CurrentSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that measures the current flowing through an electrical circuit.,https://www.theworldavatar.com/kg/ontodevice/,Current sensor
+MassFlowSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that measures the mass flow rate of gas or liquid.,https://www.theworldavatar.com/kg/ontodevice/,Mass flow sensor
 Fridge,Class,https://saref.etsi.org/core/Appliance,IS-A,,,,A device that is usually used for the preservation of items like food via temperature regulation.,https://www.theworldavatar.com/kg/ontodevice/,Fridge
+Oven,Class,https://saref.etsi.org/core/Appliance,IS-A,,,,A device that is used for heating up items like food using a heating element.,https://www.theworldavatar.com/kg/ontodevice/,Oven
+IceMaker,Class,https://saref.etsi.org/core/Appliance,IS-A,,,,A device that is used to create ice  by freezing water.,https://www.theworldavatar.com/kg/ontodevice/,Ice Maker
+Dehumidifier,Class,https://saref.etsi.org/core/Appliance,IS-A,,,,A device that reduces the humidity of the surrounding atmosphere.,https://www.theworldavatar.com/kg/ontodevice/,Dehumidifier
+WaterChiller,Class,https://saref.etsi.org/core/Appliance,IS-A,,,,A device that is used to reduce water temperature.,https://www.theworldavatar.com/kg/ontodevice/,Water chiller
+PaperShredder,Class,https://saref.etsi.org/core/Appliance,IS-A,,,,A device used to destroy paper document via cutting the document to small pieces.,https://www.theworldavatar.com/kg/ontodevice/,Paper Shredder
```

Comment:
Similarly lowercase for "Shredder" and I am not sure whether a paper shredder would be considered an appliance? Based on saref's definition, an appliance is an electrical/mechanical machine that accomplish some household functions, such as cleaning or cooking.